### ISSUE TITLE
[BUG] `logo_style` ignored from configuration

### ISSUE THREAD

**Issue by MilesTEG1** [HorlogeSkynet/archey4#128]
**Describe the bug**
On macOS, retro logo isn't shown when config.json file is used even though it's correctly set in the json file.
```json
	"logo_style": "retro",
```
But it works fine when set in the command line with `--logo-style retro`.

**Expected behavior**
When the retro logo_style is set in config.json file it should be shown with the command: 
`archey --config-path ~/.config/archey4/config.json`

**Screenshots**

![image](https://user-images.githubusercontent.com/29298312/213392740-24524172-3683-4e96-9000-41afaa3df48a.png)

![image](https://user-images.githubusercontent.com/29298312/213393295-a197ef87-2787-4fb4-ae6d-9d3b870fb319.png)

**Environment**

 - Version used : **4.14.0.1** 
 - Method of installation (distribution package, PyPI, Homebrew, sources) : **Homebrew**
 - Hardware type (laptop, server, Raspberry, hyper-visor) : **MacBook Air M2**
 - Python version (3.Y.Z) : **Python 3.9.6**
 - Operating system and version : **macOS Ventura 13.1**
 - Graphical environment name and version : **--**
 - Connectivity (off-line, LAN only, Internet access) : **LAN and Internet access**

**Additional context**

**Custom configuration**

<details><summary>Custom configuration</summary>
<p>

```json
{
	"allow_overriding": true,
	"parallel_loading": true,
	"suppress_warnings": false,
	"entries_color": "",
	"honor_ansi_color": true,
	"logo_style": "retro",
	"entries": [
		{ "type": "User" },
		{ "type": "Hostname" },
		{ "type": "Model" },
		{ "type": "Distro" },
		{
			"type": "Kernel",
			"check_version": false
		},
		{ "type": "Uptime" },
		{
			"type": "LoadAverage",
			"decimal_places": 2,
			"warning_threshold": 1.0,
			"danger_threshold": 2.0
		},
		{ "type": "Processes" },
		{ "type": "WindowManager" },
		{ "type": "DesktopEnvironment" },
		{ "type": "Shell" },
		{
			"type": "Terminal",
			"use_unicode": true
		},
		{ "type": "Packages" },
		{
			"type": "Temperature",
			"char_before_unit": " ",
			"sensors_chipsets": [],
			"sensors_excluded_subfeatures": [],
			"use_fahrenheit": false
		},
		{
			"type": "CPU",
			"one_line": false,
			"show_cores": true
		},
		{
			"type": "GPU",
			"one_line": false,
			"max_count": 2
		},
		{
			"type": "RAM",
			"warning_use_percent": 33.3,
			"danger_use_percent": 66.7
		},
		{
			"type": "Disk",
			"show_filesystems": ["local"],
			"combine_total": true,
			"disk_labels": null,
			"hide_entry_name": null,
			"warning_use_percent": 50,
			"danger_use_percent": 75
		},
		{
			"type": "LAN_IP",
			"one_line": true,
			"max_count": 2,
			"show_global": false,
			"ipv6_support": false
		},
		{
			"type": "WAN_IP",
			"one_line": true,
			"ipv4": {
				"dns_query": "myip.opendns.com",
				"dns_resolver": "resolver1.opendns.com",
				"dns_timeout": 1,
				"http_url": "https://v4.ident.me/",
				"http_timeout": 1
			},
			"ipv6": {
				"dns_query": "myip.opendns.com",
				"dns_resolver": "resolver1.opendns.com",
				"dns_timeout": 1,
				"http_url": "https://v6.ident.me/",
				"http_timeout": 1
			}
		}
	],
	"default_strings": {
		"latest": "latest",
		"available": "available",
		"no_address": "No Address",
		"not_detected": "Not detected",
		"virtual_environment": "Virtual Environment"
	}
}
```

</p>
</details>

**Comment by HorlogeSkynet** [comment]
Yes, configuring the logo style has been implemented in #124, which has not been released yet.
It will be soon !

**Comment by MilesTEG1** [comment]
> Yes, configuring the logo style has been implemented in #124, which has not been released yet. It will be soon !

Oh great 😀
Hope this will be soon, very soon :p

**Comment by HorlogeSkynet** [comment]
Here you are : https://github.com/HorlogeSkynet/archey4/releases/tag/v4.14.1.0
